apps/pesayetu/next.config.js

```diff
@@ -1,8 +1,27 @@
-const withTM = require("next-transpile-modules")([
-  "@commons-ui/core",
-  "@commons-ui/next",
-]);
+const withTM = require("next-transpile-modules")(
+  ["@commons-ui/core", "@commons-ui/next"],
+  {
+    debug: /(^|\s)+--inspect(\s|$)+/.test(process.env.NODE_OPTIONS),
+  }
+);
+
+const path = require("path");
+
+const PROJECT_ROOT = process.env.PROJECT_ROOT?.trim();
+const outputFileTracingRoot = PROJECT_ROOT
+  ? path.resolve(__dirname, PROJECT_ROOT)
+  : undefined;
 
 module.exports = withTM({
+  experimental: {
+    outputStandalone: true,
```

Comment:
shouldn't we remove `outputStandalone: true,` from experimental too?

Reply:
Yes, we should do that for all apps @esirK.